test/unit/test_config.rb

```diff
@@ -6,52 +6,52 @@
 # Unit test the Config class
 class TestConfig < MiniTest::Test
   # Just copy and paste this in from the README
-  def readme_instructions(t)
+  def readme_instructions(task)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)